exec/java-exec/src/main/java/org/apache/drill/exec/store/dfs/FileSelection.java

```diff
@@ -44,6 +45,10 @@
   public List<String> files;
   public String selectionRoot;
 
+  // this is a temporary location for the reference to Parquet metadata
+  // TODO: ideally this should be in a Parquet specific derived class.
+  public ParquetTableMetadata_v1 parquetMeta = null;
```

Comment:
Can we please make this a private field and add the appropriate getter before merging?